torchvision/ops/misc.py

```diff
@@ -16,59 +16,6 @@
 import warnings
 import torch
 from torchvision.ops import _new_empty_tensor
-from torch.nn import Module, Conv2d
-import torch.nn.functional as F
-
-
-class ConvTranspose2d(torch.nn.ConvTranspose2d):
-    """
-    Equivalent to nn.ConvTranspose2d, but with support for empty batch sizes.
-    This will eventually be supported natively by PyTorch, and this
-    class can go away.
-    """
-    def forward(self, x):
-        if x.numel() > 0:
-            return self.super_forward(x)
-        # get output shape
-
-        output_shape = [
-            (i - 1) * d - 2 * p + (di * (k - 1) + 1) + op
-            for i, p, di, k, d, op in zip(
-                x.shape[-2:],
-                list(self.padding),
-                list(self.dilation),
-                list(self.kernel_size),
-                list(self.stride),
-                list(self.output_padding),
-            )
-        ]
-        output_shape = [x.shape[0], self.out_channels] + output_shape
-        return _new_empty_tensor(x, output_shape)
-
-    def super_forward(self, input, output_size=None):
-        # type: (Tensor, Optional[List[int]]) -> Tensor
-        if self.padding_mode != 'zeros':
-            raise ValueError('Only `zeros` padding mode is supported for ConvTranspose2d')
-
-        output_padding = self._output_padding(input, output_size, self.stride, self.padding, self.kernel_size)
-
-        return F.conv_transpose2d(
-            input, self.weight, self.bias, self.stride, self.padding,
-            output_padding, self.groups, self.dilation)
-
-
-class BatchNorm2d(torch.nn.BatchNorm2d):
```

Comment:
This is technically a BC-breaking change, as if users were relying on `ConvTranspose2d` or `BatchNorm2d` from `misc.py` this will error now. I'll merge this as is to move forward, but I'll send a follow-up PR adding back a backward-compatibility fix.

Reply:
Got it. Thanks a lot @fmassa